benzina/src/array.rs

```diff
@@ -0,0 +1,256 @@
+use std::fmt::Debug;
+
+use diesel::{
+    deserialize::{FromSql, FromSqlRow},
+    expression::{AppearsOnTable, Expression, SelectableExpression},
+    pg::{Pg, PgValue},
+    query_builder::{AstPass, QueryFragment, QueryId},
+    result::QueryResult,
+    serialize::ToSql,
+    sql_types::{self, BigInt, Bool, Double, Float, Integer, Nullable, SmallInt, Text},
+};
+
+use crate::{U15, U31, U63, error::InvalidArray};
+
+/// A diesel [`Array`] serialization and deserialization wrapper _WITHOUT_ NULL items
```

Comment:
It's mostly personal preference, but I would rather use a bold style for `_WITHOUT_` instead of the uppercase.